packages/core/database/src/lifecycles/subscribers/index.ts

```diff
@@ -0,0 +1,10 @@
+import type { Subscriber } from '../types';
+
+export const isValidSubscriber = (subscriber: Subscriber) => {
```

Comment:
Won't change anything for ts as we already check the right type not unknown. 

If we consider ts in itself we could even delete it but we know this can be called from js so we still need a runtime check.